protocol/app/app.go

```diff
@@ -290,6 +290,9 @@ type App struct {
 
 	PriceFeedClient    *pricefeedclient.Client
 	LiquidationsClient *liquidationclient.Client
+	BridgeClient       *bridgeclient.Client
+
+	HealthMonitor *daemonservertypes.HealthMonitor
```

Comment:
Health Monitor does not need access to server responses, has been moved to the app.